libr/util/sys.c

```diff
@@ -905,6 +935,55 @@ R_API int r_sys_run(const ut8 *buf, int len) {
 	return ret;
 }
 
+R_API int r_sys_run_rop(const ut8 *buf, int len) {
+	const int sz = 4096;
+	int pdelta;
+#if USE_FORK
+	int st, pid;
+#endif
+	// TODO: define R_SYS_ALIGN_FORWARD in r_util.h
+	ut8 *bufptr, *p = malloc ((sz + len) << 1);
+	bufptr = p;
+	pdelta = ((size_t)(p)) & (4096 - 1);
+	if (pdelta) {
+		bufptr += (4096 - pdelta);
+	}
+	if (!bufptr || !buf) {
```

Comment:
oh well its chkd here.. which is a bit late

Reply:
:joy: my bad, working on this now :+1: